science/propka/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
+
+name                    propka
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 LGPL-2.1
```

Comment:
please add ````categories science python``` (the primary category is `science` as the Portfile resides in that directory, but the `python` PortGroup only adds `categories python`, which causes the mismatch).